.github/workflows/GAP.yml

```diff
@@ -0,0 +1,32 @@
+name: "GAP tests"
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: "GAP ${{ matrix.gap-branch }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        digraphs-lib:
+          - digraphs-lib-0.6
+        gap-branch:
+          - master
+          - stable-4.11
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gap-actions/setup-gap-for-packages@master
+        with:
+          GAP_PKGS_TO_CLONE: "wilfwilson/NautyTracesInterface"
```

Comment:
Why's this necessary? The "wilfwilson" part?

Reply:
The purpose of the action `gap-actions/setup-gap-for-packages` is to install GAP and its packages.

`NautyTracesInterface` doesn't have any releases, and so it is not installed by the action `gap-actions/setup-gap-for-packages` by default.

Therefore, the action needs to be told to install the `NautyTracesInterface` package explicitly. Writing
```
GAP_PKGS_TO_CLONE: "NautyTracesInterface"
```
would clone the master branch of `NautyTracesInterface` from the `gap-packages` repository. I intend for this to be what is written ultimately.

After cloning/installing the packages, `gap-actions/setup-gap-for-packages` then compiles the packages using GAP's `bin/BuildPackages.sh` script.

However, this currently fails for `NautyTracesInterface`, because `NautyTracesInterface` has an extra non-standard step for compiling it. (You have to first change into the nauty directory, and then compile nauty, before compiling the package.)

Since GAP 4.11, `BuildPackages.sh` looks for a `prerequisites.sh` script in each package to run before compiling the package; the `Semigroups` package has such a script, for example, to install `libsemigroups`.

`NautyTracesInterface` could use a `prerequisites.sh` script to do the step of compiling nauty, but it doesn't have one yet. I have made a pull request to `NautyTracesInterface` to do just this https://github.com/gap-packages/NautyTracesInterface/pull/40 however it has not yet been merged.

Therefore, for now, instead of pointing to `gap-packages/NautyTracesInterface`, I am pointing to my own fork, which is where the PR lives. Once the PR is merged, this line can be updated as follows:
```suggestion
          GAP_PKGS_TO_CLONE: "NautyTracesInterface"
```
I could also just compile `NautyTracesInterface` manually with an extra line or two in this workflow. However, I am trying to make this workflow as streamlined as possible.

Reply:
This now uses the `gap-packages` repository directly, the reference to my `wilfwilson` fork is gone.